test/parallel/test-fs-existssync-false.js

```diff
@@ -18,15 +18,13 @@ tmpdir.refresh();
 // Make a long path.
 for (let i = 0; i < 50; i++) {
   dir = `${dir}/1234567890`;
-  try {
-    fs.mkdirSync(dir, '0777');
-  } catch (e) {
-    if (e.code !== 'EEXIST') {
-      throw e;
-    }
-  }
 }
 
+fs.mkdirSync(dir, {
+  mode: '0777',
+  recursive: true,
+});
```

Comment:
Does the recursive option set the mode on every subfolder that is created? Or only the last one? Does it need to? Just a bit afraid that this is changing the nature of the test.

Reply:
It doesn't affect anything. The implementation is as follows:

```
while (req_wrap->continuation_data()->paths().size() > 0) {
  std::string next_path = req_wrap->continuation_data()->PopPath();
  int err = uv_fs_mkdir(loop, req, next_path.c_str(), mode, nullptr);
```